components/mediation/data-publishers/org.wso2.micro.integrator.prometheus.publisher/src/main/java/org/wso2/micro/integrator/prometheus/publisher/util/MetricFormatter.java

```diff
@@ -0,0 +1,65 @@
+package org.wso2.micro.integrator.prometheus.publisher.util;
+
+import io.prometheus.client.Collector;
+import io.prometheus.client.Collector.MetricFamilySamples;
+import io.prometheus.client.Collector.Type;
+
+import java.util.Enumeration;
+
+public class MetricFormatter {
+ private String typeString(Type type) {
+
+        switch (type) {
+            case GAUGE:
+                return "gauge";
+            case COUNTER:
+                return "counter";
+            case SUMMARY:
+                return "summary";
+            case HISTOGRAM:
+                return "histogram";
+            default:
+                return "untyped";
+        }
+    }
+
+    public StringBuilder formatMetrics(Enumeration<MetricFamilySamples> mfs) {
```

Comment:
Shall we add a doc comment to this method?

Reply:
Fixed